styles/styles.css

```diff
@@ -441,6 +444,19 @@ main img {
 }
 
 
+.icon.icon-link-blue {
```

Comment:
Can you please also remove this line as it will be redundant with your change
https://github.com/hlxsites/sunstar/blob/main/blocks/social/social.css#L106